### DISCUSSION TITLE
Need interactive mode and jupyter cell support

### DISCUSSION THREAD

**Discussion by daseindev1** [zed-industries/zed#6540] [Feature Requests]
I am a data science developer. I hope Zed will support jupyter and interactive mode as VSCode does.
For example, "# %%" can make codes below to be an interactive cell.
Appreciated very much!

**Comment by Tom-Boscher** [comment]
Something like Hydrogen Atom plugin would be a really nice integration!

https://github.com/nteract/hydrogen

**Comment by mertdeveci5** [comment]
Any ETA here?

**Comment by jansol** [comment]
#5273
